### PR TITLE
Limit zeroed memory to 64k.

### PR DESCRIPTION
**Pull request by NandkumarJoshi**

### CODE REVIEW

File: memory_map.py [hunk 1]
```diff
@@ -191,8 +191,13 @@ def consolidate_address_ranges(regions):
     for region in sorted_list:
         memories.update({region["name"] : region})
 
-    return memories
+    for _, memory in memories.items():
+        if 'memory' in memory["name"] and memory["length"] > 0x10000:
+            print("***NOTE: Only 64K of", memory["name"], "is zeroed***", file=sys.stderr)
+            print("If needed, update linker script to required value.", file=sys.stderr)
```
Comment (bsousi5) [inline memory_map.py:197]: No need to print these message out here. Adding a comment is sufficient.
Explaining for why the need to limit the scrub size.

You should however, have comment printout in the generated linker file. You would do that here https://github.com/sifive/ldscript-generator/blob/master/templates/base.lds#L89

Comment (NandkumarJoshi) [reply memory_map.py:197]: /* 'memory_0' zero scrub size is limited to 64K due to limitation in RTL sim run,
     * sizes greater than ~2GB lead to linker relocation error. */
is this comment fine enough ?

Comment (bsousi5) [reply memory_map.py:197]: /* Default memory to zero-scrub for at most 64KB, for limiting RTL simulation run time. */
/* User is recommended to enable the full size for manual RTL simulation run!                */

Comment (NandkumarJoshi) [reply memory_map.py]: Fixed.

**Pull request merged**